src/SwiftRuntimeLibrary/src/ISwiftObject.cs

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Runtime.InteropServices;
+
+namespace SwiftRuntimeLibrary
```

Comment:
What about this instead:

```suggestion
namespace Swift.Runtime
```

Reply:
Sounds good. Is it compatible with ObjC namespace naming?

Reply:
Yes, I don't think that will clash with anything.